CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
```

Comment:
Should we use this file for the changelog? https://github.com/Instawork/django-admin-fast-search/blob/master/HISTORY.rst

Currently this is the one used for publishing on readthedocs as well.

Reply:
Makes sense, made the change